modules/workspace.nix

```diff
@@ -170,5 +197,20 @@ in
         priority = 3;
       };
     })
+    (lib.mkIf (cfg.workspace.wallpaperPictureOfTheDay != null) {
+      programs.plasma.startup.desktopScript."set_wallpaper_potd" = {
+        text = ''
+          let allDesktops = desktops();
+          for (const desktop of allDesktops) {
+              var desktop = allDesktops[desktopIndex];
```

Comment:
This doesn't compile as you redefine desktop and also desktopIndex is undefined.